cmake/FindSpicy.cmake

```diff
@@ -11,17 +11,10 @@
 #
 #     If SPICY_FOUND is true:
 #
-#         SPICYC                        full path to spicyc
```

Comment:
I'm just realizing something: analyzers are pulling this in indirectly through `find_package(Spicy)` in `FindSpicyPlugin.cmake`. Could they be using any of the removed values?

Reply:
Yes, they might be. Should I drop the cleanup patch?

Reply:
Well, yes, but when building from Zeek, we then actually need to set them to appropriate values too.